environment/integration/helpers.go

```diff
@@ -169,7 +203,7 @@
 	})
 
 	if daggerErr != nil {
-		t.Skipf("Skipping test - Dagger not available: %v", daggerErr)
+		t.Fatalf("Dagger not available: %v", daggerErr)
```

Comment:
🐛 **Bug (Low):** Changed from t.Skipf to t.Fatalf but this means tests will fail instead of skip when Dagger is unavailable. This might be too aggressive for some CI environments.

```suggestion
        t.Skipf("Dagger not available (use 'go test -short' to skip): %v", daggerErr)
```